sdk/quantum/azure-quantum-jobs/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### New features
+Initial release of Azure.Quantum.Jobs. See [Getting Started](https://github.com/Azure/azure-sdk-for-java/blob/feature/azure-quantum-sdk/sdk/quantum/azure-quantum-jobs/README.md#getting-started) for more details.
```

Comment:
Replace `Azure.Quantum.Jobs` with `azure-quantum-jobs`. Also, the readme link should point to `master` branch.